components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -128,3 +128,11 @@ ok=Ok
 mandatory.claims=Mandatory claims
 configuration.error=Configuration Error!
 authentication.mechanism.not.configured=Authentication mechanism not properly configured.
+add.challenge.answers=Add Answers for Challenge Questions
```

Comment:
make all security questions to challenge questsions

Reply:
fixed